src/assets/scss/helpers/_variables.scss

```diff
@@ -10,6 +12,8 @@ $margin: 15px;
 // Fonts 
 $text-font: 'Oxygen', sans-serif;
 $highlight-font: 'Petit Formal Script', cursive;
+$logo-font: 'Milasian', cursive;
+$nueva-font: 'Bluebell', cursive;
```

Comment:
english... but is better to use a general name, something like secondary font... just if it will used on all the page...